source/mir/combinatorics/package.d

```diff
@@ -308,42 +274,13 @@ Permutations makePermutations(Allocator)(auto ref Allocator alloc, size_t n)
     return Permutations(state, indices);
 }
 
-/*
-Disposes a Permutations object. It destroys and then deallocates the
-Permutations object pointed to by a pointer.
-It is assumed the respective entities had been allocated with the same allocator.
-
-Params:
-    alloc = Custom allocator
-    perm = Permutations object
-*/
-void dispose(Allocator)(auto ref Allocator alloc, auto ref Permutations perm)
-{
-    import std.experimental.allocator: dispose;
-    dispose(alloc, perm.state);
-    dispose(alloc, perm.indices);
-}
-
-static if (__VERSION__ > 2069) @nogc unittest
-{
-    import std.algorithm: equal;
-    import std.range : iota;
-
-    import std.experimental.allocator.mallocator;
-
-    static immutable expected2 = [[0, 1], [1, 0]];
-    auto r = iota(2);
-    auto rp = makePermutations(Mallocator.instance, r.length);
-    assert(rp.indexedRoR(r).equal!equal(expected2));
-    dispose(Mallocator.instance, rp);
-}
-
+/// ditto
 struct Permutations
```

Comment:
"Forward range of Permutations?"

Or more detailed? Most of the stuff is already explained at the `permutations` method, hence the `ditto`.